pkg/object/object.go

```diff
@@ -50,16 +71,129 @@ type OpenCompose struct {
 	Volumes  []Volume
 }
 
+func (s *Service) EmptyDirVolumeExists(name string) bool {
+
+	for _, emptyDirVolume := range s.EmptyDirVolumes {
+		if name == emptyDirVolume.Name {
+			return true
+		}
+	}
+	return false
+}
+
+func (o *OpenCompose) VolumeExists(name string) bool {
+	for _, volume := range o.Volumes {
+		if name == volume.Name {
+			return true
+		}
+	}
+	return false
+}
+
+// Documentation about the valid identifiers can be found at
+// https://github.com/kubernetes/community/blob/master/contributors/design-proposals/identifiers.md
+func isValidName(name string) error {
```

Comment:
>another issue is that when I put wrong name i get just this error:
`ERROR: must match the regex [a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)* (e.g. 'example.com')`
But it doesn't say where it is. :-(
I guess this issue will be same with other validations here. :-(

done